src/test/resources/doc-assembly-template.json

```diff
@@ -19,23 +19,60 @@
   "docUploadForParty":"CLAIMANT",
   "eyewitnessUploadForParty":"DEFENDANT",
   "hearingCourtName":"Defendant Court",
-  "hearingCourtAddress":"Defendant Court address\nSW1P4BB\nLondon",
+  "hearingCourtAddress":{
+    "AddressLine1": "Defendant Court address",
+    "AddressLine2": null,
+    "AddressLine3": null,
+    "Country": null,
+    "PostCode": "SW1P4BB",
+    "PostTown": "London"
+  },
   "estimatedHearingDuration":"FOUR_HOURS",
   "hearingStatement":"No idea",
-  "otherDirectionList":[
+  "otherDirections":[
     {
       "OtherDirection":{
         "extraOrderDirection":"OTHER",
         "directionComment":"a direction",
         "sendBy":"11-10-2020",
-        "forParty":"BOTH",
-        "hmctsURL":"https://www.gov.uk/make-money-claim"
+        "forParty":"BOTH"
+      }
+    },
+    {
+      "OtherDirection":{
+        "extraOrderDirection":"EXPERT_REPORT_PERMISSION",
+        "expertReportPermissionStatementList": [
+          {
+            "id": null,
```

Comment:
Id should be UUID, never it will be null as per CCD guys. Thanks

Reply:
👍